services/grafana/client.go

```diff
@@ -300,7 +303,32 @@ func (c *Client) testDeleteUser(ctx context.Context, userID int, authHeaders htt
 	return c.do(ctx, "DELETE", "/api/admin/users/"+strconv.Itoa(userID), "", authHeaders, nil, nil)
 }
 
-func (c *Client) testCreateAPIKey(ctx context.Context, name string, role role, authHeaders http.Header) (int, string, error) {
+// CreateAdminAPIKey creates API key with Admin role and provided name.
+func (c *Client) CreateAdminAPIKey(ctx context.Context, name string) (int, string, error) {
+	headers, ok := metadata.FromIncomingContext(ctx)
+	if !ok {
+		return 0, "", fmt.Errorf("cannot get headers from metadata %v", headers)
+	}
+	// get authorization from headers.
+	authorizationHeaders := headers.Get("Authorization")
```

Comment:
maybe create a constant values for `Authorization`? Since, I see, it is used in many places.